### DISCUSSION TITLE
Can I use port 587 instead of port 25?

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#647] [Self-hosting question] [answered]
Hello there,
Is it possible to use docker and map port 25 of the container to port 465 of the real machine? Would everything work properly?
Thank you for your reply in advance.
Kind regards.

**Comment by nguyenkims** [comment, marked as answer]
@ibex107 Yes you can map any host port to container port. However SimpleLogin doesn't have any container using port 25 as far as I know.

**Reply by ghost** [reply]
I see. The docker part is my mistake but I mean it more generaly. Can you use port 465 instead of port 25 because port 25 is blocked my most isp's and cloud providers block it by default as well? Would email arrive properly?

**Reply by nguyenkims** [reply]
Ah you need to have the port 25 open in order to receive and send emails.

**Reply by ghost** [reply]
Ok. Thanks

**Comment by mrghxst** [comment]
> Ah you need to have the port 25 open in order to receive and send emails.

Is there no way around port 25 with something like a smtp relay ??

**Reply by nguyenkims** [reply]
Unfortunately no as senders contact your server at the port 25.

**Comment by mrghxst** [comment]
Thank you I didn't know that.
(New Topic Kinda)
I think I remember seeing anything that it is possible to change outgoing to the domain of my smtp relay so I dont have to manage all the problems with not getting into spam.... Would be really cool if you could tell me where I can change them If they exist. 
Thanks! love your extremely fast and helpful replies.

**Reply by nguyenkims** [reply]
Some people have managed to use AWS SES to deliver emails instead that would avoid you having to manage the server reputation. The doc is on https://github.com/simple-login/app/blob/master/docs/ses.md, hope it can be useful :)
